src/CoreBundle/Entity/Company.php

```
@@ -51,6 +70,67 @@
     #[Assert\NotBlank()]
     public ?string $currency = '';
 
+    // Related entities: Only added here to enable orphan removal
+    #[ORM\OneToMany(mappedBy: 'company', targetEntity: ApiTokenHistory::class, cascade: ['persist'], orphanRemoval: true)]
+    private Collection $apiTokenHistories;
```

PHPMD (2.15.0)

Avoid unused private fields such as '$apiTokenHistories'. (Unused Code Rules)

(UnusedPrivateField)